.travis.yml

```diff
@@ -96,3 +96,10 @@ matrix:
       script:
       - "scripts/build_assembly_no_test.sh scalding-core"
       - "scripts/test_typed_tutorials.sh"
+
+    - scala: 2.10.4
+      env: BUILD="test execution tutorials"
+      script:
+      - "scripts/build_assembly_no_test.sh scalding-core"
```

Comment:
Yup, have removed this and merged with most recent develop. It's
merge-green now.

On Tue, Feb 17, 2015 at 4:17 PM, P. Oscar Boykin <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/twitter/scalding/pull/1196#discussion_r24868820>:
>
> > @@ -96,3 +96,10 @@ matrix:
> >        script:
> >        - "scripts/build_assembly_no_test.sh scalding-core"
> >        - "scripts/test_typed_tutorials.sh"
> > +
> > +    - scala: 2.10.4
> > +      env: BUILD="test execution tutorials"
> > +      script:
> > +      - "scripts/build_assembly_no_test.sh scalding-core"
>
> do we still need this with the fixes you put in place in the bash script?
> I'm trying to minimize the time the tests take here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/scalding/pull/1196/files#r24868820>.
>